src/feature-flags.ts

```diff
@@ -0,0 +1,45 @@
+/** All registered GK togglers. Do not manually add items to it. */
+const GateKeeperTogglers: Record<string, () => void> = {};
+
+// Register your new feature flags below, both in type and in `registerFeatureFlagChecker`.
+type FeatureFlagName = 'RequirementDebugger' /* | 'AddYourFeatureFlagNameHere' */;
+const featureFlagCheckers: FeatureFlagCheckers = registerFeatureFlagChecker(
+  'RequirementDebugger'
+  /* 'AddYourFeatureFlagNameHere' */
+);
+export default featureFlagCheckers;
+
+// Add your new feature flags above
+// Gate keeper runtime code below. Do not touch if you do not understand.
+
+type FeatureFlagCheckers = Record<`is${FeatureFlagName}Enabled`, () => boolean>;
```

Comment:
Using the feature from TS 4.1: https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html